CHANGES.rst

```diff
@@ -8,16 +8,19 @@ Unreleased
 
 * :meth:`.LiveContributorRelationship.leave` to abdicate the live thread
   contributor position.
+* :meth:`.LiveContributorRelationship.remove` to remove the redditor
+  from the live thread contributors.
 * :attr:`.limits` to provide insight into number of requests made and remaining
   in the current rate limit window.
+* :meth:`.LiveThreadContribution.add_update` to add an update to the live
+  thread. :class:`.LiveThreadContribution` instance can be retrieved
+  through ``thread.contrib``.
 
 **Fixed**
 
 * :meth:`.LiveContributorRelationship.invite` and
   :meth:`.LiveContributorRelationship.remove_invite` now hit endpoints,
   which starts with 'api/', for consistency.
-* :meth:`.LiveContributorRelationship.remove` to remove the redditor
-  from the live thread contributors.
```

Comment:
lines 19-20 are moved to correct 'Added' section. I misplaced these in previous PR.